Makefile

```diff
@@ -56,6 +56,10 @@ black:
 	# check that black formatting would not be applied
 	black --check .
 
+check_pre_commit_hooks:
```

Comment:
please also add next target: 
```
docker_check_pre_commit_hooks: build
    # check if all pre-commit hooks are passing inside ibis container
    $(DOCKER_RUN) pre-commit run --all-files
```
So we will be able to use one target in CI and locally.